wordpress-mobile/WordPress-Android
# Issue/10651 material notification details

Part of #10651

#11192 needs to be merged first.

This PR moves notification details to the material theme and adds dark mode support.

Notifications are tricky, and I'm not sure about visual changes :) 

![comparison_notes](https://user-images.githubusercontent.com/728822/73602106-e025ce00-4522-11ea-97c6-9754535cbfec.png)

To test:
- Check details of different types of notifications and make sure they look ok, and nothing stands out as weird, in both light and dark themes.

You can change the app theme from the App Settings.


PR submission checklist:
